snapshot/cli/cli.go

```diff
@@ -119,3 +127,36 @@ func (c *ingestGitCommitCommand) run(db snapshot.DB, _ *cobra.Command, _ []strin
 	fmt.Println(id)
 	return nil
 }
+
+type exportGitCommitCommand struct {
+	id string
+}
+
+func (c *exportGitCommitCommand) register() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "export_git_commit",
```

Comment:
From store? It's not from store; this is how you get something into scoot or out at all...